### ISSUE TITLE
Aerojump fails to launch

### ISSUE THREAD

**Issue by pehrnord** [philip-karlsson/aerojump.nvim#1]
Using the space-key with:
nmap <space> <Plug>(AerojumpSpace)
I get:
```
Error detected while processing function remote#define#request:                                                                                                                                                                                
line    2:                                                                                                                                                                                                                                     
error caught in request handler '/home/H743800/.local/shared/nvim/plugged/aerojump.nvim/rplugin/python3/aerojump:command:Aerojump [['kbd', 'space'], [320, 320]]':                                                                             
Traceback (most recent call last):                                                                                                                                                                                                             
  File "/home/H743800/.local/shared/nvim/plugged/aerojump.nvim/rplugin/python3/aerojump/__init__.py", line 257, in Aerojump                                                                                                                    
    self.__draw()                                                                                                                                                                                                                              
  File "/home/H743800/.local/shared/nvim/plugged/aerojump.nvim/rplugin/python3/aerojump/__init__.py", line 98, in __draw                                                                                                                       
    self.__set_top_pos(self.top_pos)                                                                                                                                                                                                           
  File "/home/H743800/.local/shared/nvim/plugged/aerojump.nvim/rplugin/python3/aerojump/__init__.py", line 67, in __set_top_pos                                                                                                                
    self.nvim.current.window.cursor = top_pos                                                                                                                                                                                                  
  File "/usr/lib/python3.7/site-packages/pynvim/api/window.py", line 27, in cursor                                                                                                                                                             
    return self.request('nvim_win_set_cursor', pos)                                                                                                                                                                                            
  File "/usr/lib/python3.7/site-packages/pynvim/api/common.py", line 51, in request                                                                                                                                                            
    return self._session.request(name, self, *args, **kwargs)                                                                                                                                                                                  
  File "/usr/lib/python3.7/site-packages/pynvim/api/nvim.py", line 182, in request                                                                                                                                                             
    res = self._session.request(name, *args, **kwargs)                                                                                                                                                                                         
  File "/usr/lib/python3.7/site-packages/pynvim/msgpack_rpc/session.py", line 102, in request                                                                                                                                                  
    raise self.error_wrapper(err)                                                                                                                                                                                                              
pynvim.api.nvim.NvimError: b'Cursor position outside buffer'                                                                                                                                                                                   
```

**Comment by ripxorip** [comment]
Where were the cursor when you summoned the plugin? Last line? Thx for reporting

**Comment by ripxorip** [comment]
Might have been solved by one of the two previous commits :)

**Comment by ripxorip** [comment]
Solved by #3
